### ISSUE TITLE
error when creating "STDIN"....Snippet directives are disabled by the Ingress administrator

### ISSUE THREAD

**Issue by phughesion** [SpecterOps/Nemesis#9]
I am attempting to set up nemesis for the first time. I am not too familiar with Kubernetes, so perhaps this is a common error. Googling has led me to believe it may be an issue with the .yaml file. However, I have not modified skaffold.yaml so perhaps there is a bug there.

I started with a fresh install of Debian and followed the setup line by line. Please let me know if any additional info would assist in the resolution of this issue.

```
Generating tags...
 - nemesis-dotnet -> nemesis-dotnet:10470eb238e7aec743cc3fc6bf1e38b7656166252c28c0f818be1825e6b46c01
 - nemesis-nlp -> nemesis-nlp:6c0759a8213f2588ed46ed55232c2c16502064763df4d6b59881adff0ea77d65
 - nemesis-passwordcracker -> nemesis-passwordcracker:04132f3a8e6840667ae3aa379a68a420c3c0ad706f9a084450448f4c5ef50be2
 - tensorflow-serving -> tensorflow-serving:7448c93455e4df4c8c67b00b6350af5229b15e450150469a6bb91983045e8963
 - enrichment -> enrichment:2d0f7f2b87613bdd7db6f2475ddb1b9eb021d05b0bdae4a0e6d817ccdb8aa24a
 - dashboard -> dashboard:f10b6fa2ab75decd7ac41f4e840701456c68bba379b9ab20bafee76963e95cc7
Checking cache...
 - nemesis-dotnet: Found Locally
 - nemesis-nlp: Found Locally
 - nemesis-passwordcracker: Found Locally
 - tensorflow-serving: Found Locally
 - enrichment: Found Locally
 - dashboard: Found Locally
Starting test...
Tags used in deployment:
 - nemesis-dotnet -> nemesis-dotnet:180062b33f54b8634cfa41264774b1df17e0bd46be83188cdc09cda9be2f64cb
 - nemesis-nlp -> nemesis-nlp:03931c0c9ff847e782a6e7f71fafe20ac8069091f68dbcdf706c42f695c4957d
 - nemesis-passwordcracker -> nemesis-passwordcracker:6bb4122823735057f9446fdf78d1b1be5812c84ab862014d3e623117de953b2f
 - tensorflow-serving -> tensorflow-serving:2151cd9bc22cf56b2b77ddf97f92a44f7a260dd3bf562be7997ecdd5c86ea461
 - enrichment -> enrichment:99a58dde939d21c694d9111ad543f7c412c697732ac5cc4b14c37ea94f972f8f
 - dashboard -> dashboard:9561bbc1e64bb698e57ca75807d4da74a5eadf0da767a6324885682fa78e5236
Starting deploy...
 - elasticsearch.elasticsearch.k8s.elastic.co/nemesis unchanged
 - persistentvolume/elasticsearch-data-pv unchanged
 - persistentvolumeclaim/elasticsearch-data-pvc unchanged
 - ingress.networking.k8s.io/kibana-ingress unchanged
 - Warning: path /elastic(/|$)(.*) cannot be used with pathType Prefix
 - kibana.kibana.k8s.elastic.co/nemesis unchanged
 - job.batch/kibana-init-job configured
 - configmap/kibana unchanged
 - Error from server (BadRequest): error when creating "STDIN": admission webhook "validate.nginx.ingress.kubernetes.io" denied the request: nginx.ingress.kubernetes.io/configuration-snippet annotation cannot be used. Snippet directives are disabled by the Ingress administrator
kubectl apply: exit status 1
```

Error is at the very end:

**Error from server (BadRequest): error when creating "STDIN": admission webhook "validate.nginx.ingress.kubernetes.io" denied the request: nginx.ingress.kubernetes.io/configuration-snippet annotation cannot be used. Snippet directives are disabled by the Ingress administrator
kubectl apply: exit status 1**

**Comment by leechristensen** [comment]
Something changed very recently with the underlying nginx ingress, so working on a fix in the next couple hours!

**Comment by leechristensen** [comment]
Fixed now as of #10 .  pgadmin/rabbitmqadmin interfaces are slightly broken, but that's only an issue if you're doing dev. Easiest workaround for those (if it's a concern) is to forward the ports to those services yourself using `kubectl port-forward`, or modify skaffold's config to forward the ports for you.
